PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessClusterDestoryDialog.java

```diff
@@ -0,0 +1,106 @@
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * <p/>
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ * <p/>
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.microsoft.azure.sparkserverless.serverexplore.ui;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.DialogWrapper;
+import com.microsoft.azure.hdinsight.common.mvc.SettableControl;
+import com.microsoft.azure.sparkserverless.serverexplore.SparkServerlessClusterDestoryCtrlProvider;
+import com.microsoft.azure.sparkserverless.serverexplore.SparkServerlessClusterDestoryModel;
+import com.microsoft.intellij.rxjava.IdeaSchedulers;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.sparkserverless.SparkServerlessClusterNode;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.awt.event.*;
+
+public class SparkServerlessClusterDestoryDialog extends DialogWrapper
+        implements SettableControl<SparkServerlessClusterDestoryModel> {
+
+    @NotNull
+    private SparkServerlessClusterDestoryCtrlProvider ctrlProvider;
+
+    @NotNull
+    private SparkServerlessClusterNode clusterNode;
+
+    private JPanel destroyDialogPanel;
+    private JTextField clusterNameField;
+    private JTextField errorMessageField;
+
+    public SparkServerlessClusterDestoryDialog(@NotNull SparkServerlessClusterNode clusterNode) {
+        super((Project) clusterNode.getProject(), true);
+        this.ctrlProvider = new SparkServerlessClusterDestoryCtrlProvider(
+                this, new IdeaSchedulers((Project) clusterNode.getProject()));
+        this.clusterNode = clusterNode;
+
+        init();
+        this.setTitle("Delete Spark Cluster");
+        errorMessageField.setBackground(this.destroyDialogPanel.getBackground());
+        errorMessageField.setBorder(BorderFactory.createEmptyBorder());
+        this.setModal(true);
+    }
+
+    public void getData(@NotNull SparkServerlessClusterDestoryModel data) {
+        data.setClusterName(clusterNameField.getText())
+                .setErrorMessage(errorMessageField.getText());
+    }
+
+    public void setData(@NotNull SparkServerlessClusterDestoryModel data) {
+        clusterNameField.setText(data.getClusterName());
+        errorMessageField.setText(data.getErrorMessage());
+    }
+
+    @Override
+    protected void doOKAction() {
+        ctrlProvider
```

Comment:
Need to disable OK button to avoid multiple clicks, refer to new `AddNewClusterFrom::doOKAction()` please